gapis/gfxapi/sync/sync.go

```diff
@@ -41,6 +41,9 @@ type SynchronizedAPI interface {
 	// ResolveSynchronization resolve all of the synchronization information for
 	// the given API
 	ResolveSynchronization(ctx context.Context, d *Data, c *path.Capture) error
+
+	// MutateSubcommands mutates the given Atom, but calls callbackafter each subcmomand is executed
```

Comment:
`but` seems a little out of place, as it isn't contradicting anything. How about:

`// MutateSubcommands mutates the given Atom calling callback after each subcommand is executed.`

(typos also corrected)